src/frontend/src/btc/components/send/BtcSendReview.svelte

```diff
@@ -46,9 +65,11 @@
 
 	<BtcUtxosFee slot="fee" bind:utxosFee {networkId} {amount} />
 
-	<BtcSendWarnings
-		slot="info"
-		{utxosFee}
-		pendingTransactionsStatus={$hasPendingTransactionsStore}
-	/>
+	<svelte:fragment slot="info">
```

Comment:
Should we add some fade effect?

Reply:
Both components already have animations in place: `InsufficientFundsForFee` uses `slide`, and `BtcSendWarnings` - `fade`. I think it's something to be aligned when implementing the new Send flow.

Reply:
Coolio 👍